cmd/consistency/main.go

```diff
@@ -0,0 +1,132 @@
+	logInfoFileName          = "logInfo.txt"
+	outputIdentitiesFileName = "identities.txt"
+)
+
+// This main function performs a periodic root hash consistency check.
+// Upon starting, any existing latest snapshot data is loaded and the function runs
+// indefinitely to perform consistency check for every time interval that was specified.
+func main() {
+	// Command-line flags that are parameters to the verifier job
+	configFilePath := flag.String("config-file", "", "Name of the file containing the consistency check workflow configuration settings")
+	configString := flag.String("config-string", "", "Consistency check workflow configuration settings input as a string")
+	once := flag.Bool("once", false, "Perform consistency check once and exit")
+	logInfoFile := flag.String("file", "", "path to log info file")
+	userAgentString := flag.String("user-agent", "", "details to include in the user agent string")
+	flag.Parse()
+
+	if *configFilePath == "" && *configString == "" {
+		log.Fatalf("empty configuration input")
+	}
+
+	if *configFilePath != "" && *configString != "" {
+		log.Fatalf("only input one of --config-file or --config-string")
+	}
+
+	var config ConsistencyCheckConfiguration
+	if *configString != "" {
+		if err := yaml.Unmarshal([]byte(*configString), &config); err != nil {
+			log.Fatalf("error parsing identities: %v", err)
+		}
+	}
+
+	if *configFilePath != "" {
+		readConfig, err := os.ReadFile(*configFilePath)
+		if err != nil {
+			log.Fatalf("error reading from identity monitor configuration file: %v", err)
+		}
+		if err := yaml.Unmarshal([]byte(readConfig), &config); err != nil {
+			log.Fatalf("error parsing identities: %v", err)
+		}
+	}
+
+	if config.ServerURL == "" {
+		config.ServerURL = publicRekorServerURL
+	}
+
+	if config.Interval == nil {
+		defaultInterval := time.Hour
+		config.Interval = &defaultInterval
+	}
+
+	if config.OutputIdentitiesFile == "" {
+		config.OutputIdentitiesFile = outputIdentitiesFileName
+	}
+
+	if logInfoFile == nil {
+		defaultLogInfoFile := logInfoFileName
+		logInfoFile = &defaultLogInfoFile
+	}
+
+	rekorClient, err := client.GetRekorClient(config.ServerURL, client.WithUserAgent(strings.TrimSpace(fmt.Sprintf("rekor-monitor/%s (%s; %s) %s", version.GetVersionInfo().GitVersion, runtime.GOOS, runtime.GOARCH, *userAgentString))))
+	if err != nil {
+		log.Fatalf("getting Rekor client: %v", err)
+	}
+
+	verifier, err := rekor.GetLogVerifier(context.Background(), rekorClient)
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	err = rekor.VerifyConsistencyCheckInputs(config.Interval, logInfoFile, &config.OutputIdentitiesFile, once)
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	ticker := time.NewTicker(*config.Interval)
+	defer ticker.Stop()
+
+	// Loop will:
+	// 1. Fetch latest checkpoint and verify
+	// 2. If old checkpoint is present, verify consistency proof
+	// 3. Write latest checkpoint to file
+
+	// To get an immediate first tick
+	for ; ; <-ticker.C {
```

Comment:
Did you want to include a loop here? `RunConsistencyCheck` includes a loop.

I would suggest renaming the file with `git mv` and then making changes, which I believe will properly show the diff in github.

Reply:
Removing loop from within `RunConsistencyCheck` tried renaming via `git mv`, but diff doesn't still seem to be tracked- might be because I already renamed without using `git mv`